frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -291,7 +313,11 @@ export const Dashboard = () => {
           showSettingsLink={true}
         />
       </StyledHeaderMenu>
-      <CourseGrid courses={filteredSortedCourses} />
+      {tableView ? (
+        <CourseGrid courses={filteredSortedCourses} />
+      ) : (
+        <CourseTable courses={filteredSortedCourses}></CourseTable>
+      )}
```

Comment:
Fix the logic here: I'm assuming the table should show if `tableView` is true, but right now this is the opposite.

Also make `CourseTable` be a single component without the closing.